tutorial.md

```diff
@@ -0,0 +1,99 @@
+# Tutorial
+This is a tutorial to get started with the Cryosphere Virtual Laboratory (CVL).
+
+## What is CVL
+CVL stands for Cryosphere Virtual Laboratory. It offers users access to the following tools:
+1. Data portal
+The data catalogue on the CVL websites allows users to browse metadata from various data publishers to locate data from multiple sources.
+2. Computing environment
+In collaboration with Polar Thematic Exploitation Platform (PTEP) a virtual computing environment (JupyterLab) is offered. This allows users to use virtual ressources instead of local machines.
+3. 3D Viewer
+The 3D Visualiser is a tool to illustrate user projects.
+
+## Workflow overview
+The workflow is git based and uses GitHub as sharing platform.
+
+[TODO diagram]
+
+## How to get and set up required accounts 
+Accounts for GitHub, PTEP and CVL are required to follow the workflow.
+
+### GitHub
+1. Log in to https://github.com/ (or register as new user)
+2. Navigate to https://github.com/CryosphereVirtualLab/public-notebooks
+3. Fork the repository, this will create a linked private repository at the following path (where YOUR-NAME is replaced by your username): https://github.com/YOUR-NAME/public-notebooks
```

Comment:
A fork is user copy of the original repository. Once the user is happy with their changes, a pull request could be submitted to the original repository.